ci/pr-pipeline.yml

```diff
@@ -0,0 +1,97 @@
+---
+
+var:
+
+  instana-operator-git-repo-config: &instana-operator-git-repo-config
+    uri: https://github.com/instana/instana-agent-operator.git
+    username: ((project-berlin-gh-token))
+    password: x-oauth-basic
+    branch: &build-branch ((branch))
+resources:
+
+  - name: pipeline-source
+    type: git
+    icon: github
+    source:
+      <<: *instana-operator-git-repo-config
+
+jobs:
+
+  - name: self-update
+    max_in_flight: 1
+    plan:
+      - get: pipeline-source
+        trigger: true
+      - set_pipeline: self
+        file: pipeline-source/ci/pr-pipeline.yml
+        vars:
+          branch: *build-branch
+          project-berlin-gh-token: ((project-berlin-gh-token))
```

Comment:
Would it be possible to use this token for putting gh status? If we don't have to generate some new tokens and reuse this one, we can also implement putting the git status as part of this PR